.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - tiddlywiki-com
+env:
+  COMMIT_CONTEXT: "GitHub build"
+  COMMIT_EMAIL: "actions@github.com"
+  COMMIT_NAME: "GitHub Actions"
+  NODE_VERSION: "12"
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "${{ env.NODE_VERSION }}"
+      - run: "./bin/test.sh"
+  build-prelease:
```

Comment with a suggestion block to fix the prelease typo
```suggestion
  build-prerelease:
```